chart/values.yaml

```diff
@@ -3,6 +3,7 @@ image:
   tag: latest
 
 federation:
+  configMode: mcp
```

Comment:
A default for this is specified in `templates/deployment.yaml`:
```
- '--configMode={{ .Values.federation.configMode | default "mcp" }}'
```

Do we want to remove the default in `templates/deployment.yaml` and have users explicitly specify it if providing their own `values.yaml`?

Reply:
Thanks, I forgot to remove this default value. The main reason why I added `configMode` to helm values is the conditions for annotations. If I didn't add `configMode` to values, I would have to change that condition to:
```
{{- if ne .Values.configMode "k8s" }}
```
And I thought that it will be cleaner if the default value will be defined in the default values.yaml and the condition will be simpler.